src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2020 IEXEC BLOCKCHAIN TECH
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<configuration>
+
+    <!-- Default -->
+    <property name="LOG_LEVEL" value="${IEXEC_LOG_LEVEL:-INFO}"/>
+    <include resource="org/springframework/boot/logging/logback/base.xml"/>
+    <logger name="org.springframework.web" level="${LOG_LEVEL}"/>
+
+    <!-- Web3J -->
+    <!-- <logger name="org.web3j.protocol" level="DEBUG"/> -->
+
+    <!-- Feign -->
+    <!-- Enable logging for now -->
+    <logger name="feign" level="DEBUG"/>
```

Comment:
Change this level before merging ?

Reply:
Sure, see https://github.com/iExecBlockchainComputing/iexec-sms/pull/117/commits/ae2aa639aca1364efb75539e0def1924e99f0fa3.